espnet2/samplers/build_batch_sampler.py

```diff
@@ -85,8 +85,10 @@ def build_batch_sampler(
     """Helper function to instantiate BatchSampler.
 
     Args:
-        type: mini-batch type. "unsorted", "sorted", "folded", "numel", or, "length"
-        batch_size: The mini-batch size. Used for "unsorted", "sorted", "folded" mode
+        type: mini-batch type. "unsorted", "sorted", "folded", "numel",
+            "length", or "catbel"
```

Comment:
can you add an explanation about `catbel` to https://espnet.github.io/espnet/espnet2_training_option.html#change-mini-batch-type
?

Reply:
Sure, I just did. Let me know if you want supplement.